xbatcher/tests/test_generators.py

```diff
@@ -115,6 +115,21 @@ def test_batch_1d_concat(sample_ds_1d, input_size):
         assert "x" in ds_batch.coords
 
 
+def test_batch_1d_concat_duplicate_dim(sample_ds_1d):
+    """
+    Test batch generation for a 1D dataset using ``concat_input_dims`` when
+    the same dimension occurs in ``input_dims`` and `batch_dims``
+    """
+    bg = BatchGenerator(
+        sample_ds_1d, input_dims={"x": 5}, batch_dims={"x": 10}, concat_input_dims=True
+    )
+    validate_generator_length(bg)
+    expected_dims = get_batch_dimensions(bg)
+    for n, ds_batch in enumerate(bg):
```

Comment:
Remove unused `enumarate()` since the index `n` isn't used 

```suggestion
    for ds_batch in bg:
```